src/layouts/components/Editable/Editable.tsx

```diff
@@ -89,13 +90,24 @@ export const EmptySection = ({
   )
 }
 
-export type EditableSidebarProps = SidebarHeaderProps
+export type EditableSidebarProps = SidebarHeaderProps & FlexProps
 const EditableSidebar = ({
   title,
   children,
-}: PropsWithChildren<EditableSidebarProps>) => {
+  ...rest
+}: EditableSidebarProps) => {
   return (
-    <Flex flexDir="column" bg="base.canvas.alt">
+    <Flex
+      flexDir="column"
+      bg="base.canvas.alt"
+      width="450px"
+      overflowY="scroll"
+      pb="100px"
+      // NOTE: We reserve 80px **each** for
+      // both the header and the footer
+      h="calc(100vh - 160px - 1rem)"
```

Comment:
whats the 1rem for?

Reply:
good question - i'm not entirely sure too. i've taken the styling from the existing editor styles [here](https://github.com/isomerpages/isomercms-frontend/blob/develop/src/styles/isomer-cms/pages/Editor.module.scss#L98). 

i didn't investigate this because it was tangential to the feature

Reply:
was just weird to see all 3 units in same expression hahaha! but since we alr had this before i think is okay to keep it